### PR TITLE
horoscope now working properly

### PR DESCRIPTION
**Pull request by dcrystalj**
no more error:  can't compare offset-naive and offset-aware datetimes

### CODE REVIEW

File: piplmesh/panels/horoscope/panel.py [hunk 1]
```diff
@@ -53,7 +54,7 @@ def get_context(self, context):
             })
             return context
 
-        if datetime.datetime.now() > horoscope.date + datetime.timedelta(days=HOROSCOPE_OBSOLETE):
+        if datetime.datetime.utcnow().replace(tzinfo=utc) > horoscope.date + datetime.timedelta(days=HOROSCOPE_OBSOLETE):
```
Comment (mitar) [inline piplmesh/panels/horoscope/panel.py:57]: Aaa. Noo. What, we had datetime.now??? How I missed this.

The solution should be:

```
from django.utils import timezone

timezone.now()
```

You do not have to complicate with replace of timezone.

Comment (mitar) [inline piplmesh/panels/horoscope/panel.py:57]: And import only modules!

**Comment by mitjap** [comment]
Thank you for fixing that :)

**Comment by mitar** [comment]
So, does it work now also with my changes?

**Comment by dcrystalj** [comment]
works like a charm =)

**Pull request merged**